### PR TITLE
Ceph provisioning wrapper

### PR DESCRIPTION
**Pull request by cybernth**
Added the specification for create ceph-instller wrapper for tendrl
tendrl-bug-id:Tendrl/specifications/issues#48

Signed-off-by: Nishanth Thomas <nthomas@redhat.com>

### CODE REVIEW

File: specs/ceph_installer_wrapper.adoc [hunk 1]
```diff
@@ -0,0 +1,130 @@
+// vim: tw=79
+
+= Tendrl Ceph-installer Wrpper
```
Comment (mkudlej) [inline specs/ceph_installer_wrapper.adoc]: Typo: Wrapper

File: specs/ceph_installer_wrapper.adoc [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Ceph-installer allows the administrator tp provision and control the deployment
```
Comment (mkudlej) [inline specs/ceph_installer_wrapper.adoc]: Typo: to provision

File: specs/ceph_installer_wrapper.adoc [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+of Ceph clusters via a ResyFul API. Tendrl expected to make use of these APIs for provisioning the Ceph clusters.
```
Comment (mkudlej) [inline specs/ceph_installer_wrapper.adoc]: Typo: RESTFul

File: specs/ceph_installer_wrapper.adoc [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+== Problem description
+
+Ceph-installer provides a set of Rest APIs which allows one to install and configure Ceph.Ceph-installer make use of ceph-ansible (ansible based deployment tool for provisioning ceph) internaly and exposes the Rest APIs that accepts and returns JSON responses with meaningful error codes and messges.  Tendrl is expected to perform Ceph deployment and planning to leverage the capabilities of ceph-installer by writing a wrapper for integration.
```
Comment (mkudlej) [inline specs/ceph_installer_wrapper.adoc]: Typo: messages

File: specs/ceph_installer_wrapper.adoc [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+== Use Cases
```
Comment (mkudlej) [inline specs/ceph_installer_wrapper.adoc]: How does Tendrl do OSD replacing, node replacing?

**Review by mkudlej — CHANGES_REQUESTED** [review]
Please solve small typos and answer one question. Otherwise it looks good to me.

File: specs/ceph_installer_wrapper.adoc (hunk 1 above)
Comment (cybernth) [reply specs/ceph_installer_wrapper.adoc]: ok

File: specs/ceph_installer_wrapper.adoc (hunk 2 above)
Comment (cybernth) [reply specs/ceph_installer_wrapper.adoc]: ok

File: specs/ceph_installer_wrapper.adoc (hunk 3 above)
Comment (cybernth) [reply specs/ceph_installer_wrapper.adoc]: ok

File: specs/ceph_installer_wrapper.adoc (hunk 4 above)
Comment (cybernth) [reply specs/ceph_installer_wrapper.adoc]: ok

**Comment by cybernth** [comment]
> How does Tendrl do OSD replacing, node replacing?

This is a wrapper to ceph-installer which does not support the above said functionalities. If tendrl to provide  this, it should be done by different means like ansible or something else which is not in the scope of this spec

**Pull request closed without merging**